tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/tmf/core/model/tree/TmfTreeModel.java

```diff
@@ -118,6 +125,20 @@ public List<T> getEntries() {
         return fScope;
     }
 
+    /**
+     * @return
+     */
+    public int getExpandLevel() {
+        return fExpandLevel;
+    }
+
+    /**
+     * @param expandLevel
```

Comment:
change java doc

```java
	/**
	 * Sets the auto-expand level. The value 0 means that there is no
	 * auto-expand; 1 means that the invisible root element is expanded (since
	 * most concrete subclasses do not show the root element, there is usually
	 * no practical difference between using the values 0 and 1); 2 means that
	 * top-level elements are expanded, but not their children; 3 means that
	 * top-level elements are expanded, and their children, but not
	 * grandchildren; and so on.
	 * <p>
	 * The value <code>ALL_LEVELS</code> means that all subtrees should be
	 * expanded.
	 * @since 9.6
	 */
```